pages/super_search_page.py

```diff
@@ -47,16 +47,17 @@ def wait_for_page_to_load(self):
 
     def select_field(self, field):
         self.find_element(*self._input_locator).send_keys(field)
-        self.find_element(*self._operator_test_locator).click()
+        self.find_element(*self._highlighted_text_locator).click()
 
-    def select_operator(self, operator):
-        self.find_element(*self._second_input_locator).send_keys(operator)
-        self.find_element(*self._operator_test_locator).click()
+    def select_operator(self, line_id, operator):
+        input_locator = (self._operator_field_locator[0], self._operator_field_locator[1] % line_id)
```

Comment:
Should this input_locator be defined/set in the appropriate class, instead of here?